README.rst

```diff
@@ -82,6 +82,21 @@ You can also run linting with ``tox``:
     $ tox -e lint
 
 
+Releasing a new version
+------------------------
+
+``mozilla-django-oidc`` releases are hosted in `PyPI <https://pypi.python.org/pypi/mozilla-django-oidc>`_.
+Here are the steps you need to follow in order to push a new release:
+
+* Make sure that ``HISTORY.rst`` is up-to-date focusing mostly on backwards incompatible changes
+* Bump the project version and create a commit for that
```

Comment:
Oh I get it. There's a tool called "bumpversion" can that does that edit. It's in that requirements_dev.txt.

I tried to use it but perhaps I don't know how:
```
▶ bumpversion 0.3.1
Traceback (most recent call last):
  File "/Users/peterbe/virtualenvs/mozilla-django-oidc/bin/bumpversion", line 11, in <module>
    sys.exit(main())
  File "/Users/peterbe/virtualenvs/mozilla-django-oidc/lib/python3.5/site-packages/bumpversion/__init__.py", line 994, in main
    vcs.commit(message=commit_message)
  File "/Users/peterbe/virtualenvs/mozilla-django-oidc/lib/python3.5/site-packages/bumpversion/__init__.py", line 73, in commit
    list(os.environ.items()) + [(b'HGENCODING', b'utf-8')]
  File "/opt/boxen/homebrew/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/subprocess.py", line 629, in check_output
    **kwargs).stdout
  File "/opt/boxen/homebrew/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/subprocess.py", line 711, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'commit', '-F', '/var/folders/1x/2hf5hbs902q54g3bgby5bzt40000gn/T/tmpal20r8pj']' returned non-zero exit status 1
```